src/twisted/test/test_ssl.py

```diff
@@ -271,11 +271,13 @@ def __init__(self, *args, **kw):
 
 
 
-class StolenTCPTests(ProperlyCloseFilesMixin, unittest.TestCase):
+class StolenTCPTests(ProperlyCloseFilesMixin, TestCase):
     """
     For SSL transports, test many of the same things which are tested for
     TCP transports.
     """
+    if interfaces.IReactorSSL(reactor, None) is None:
```

Comment:
Should this (and the later instances in this file) be using `@skipIf` ?  (If not, I think the old version, where it's clear that a single conditional is causing multiple test classes to be skipped in the same way, is clearer)